backend/dtos/dtos.go

```diff
@@ -47,12 +47,31 @@ type Finding struct {
 	OccurredTo    *time.Time `json:"occurredTo"`
 }
 
+type TopContrib struct {
+	Slug        string `db:"slug" json:"slug"`
+	OperationID int64  `db:"operation_id" json:"operationId"`
+	Count       int64  `db:"count" json:"count"`
+}
+
+type EvidenceCount struct {
+	OperationID    int64 `db:"operation_id" json:"operationId"`
```

Comment:
Same here. Also, since these are inside an operation, you might think to just exclude these altogether (assuming you _only_ get them as part of an operation -- still going through the code, so idk)

Reply:
yeah I think I can do that, but will double check

Reply:
removed